requests/pycrystal-reset.yml

```diff
@@ -0,0 +1,3 @@
+action: token_reset
```

Comment:
This is the wrong request. You need to do
https://github.com/conda-forge/admin-requests/blob/9c8bcfedb0567d344ce43068c667831928c61d54/examples/example-add-feedstock-output.yml#L1